src/edu/csus/ecs/pc2/ui/ShadowCompareScoreboardPane.java

```diff
@@ -0,0 +1,545 @@
+        }
+        return rigidArea;
+    }
+    private Component getRigidArea_1() {
+        if (rigidArea_1 == null) {
+        	rigidArea_1 = Box.createRigidArea(new Dimension(30, 20));
+        }
+        return rigidArea_1;
+    }
+    private Component getRigidArea_2() {
+        if (rigidArea_2 == null) {
+        	rigidArea_2 = Box.createRigidArea(new Dimension(20, 20));
+        }
	rigidArea_2.setPreferredSize(new Dimension(20, 40));
+        
+        return rigidArea_2;
+    }
+    private Component getRigidArea_3() {
+        if (rigidArea_3 == null) {
+            rigidArea_3 = Box.createRigidArea(new Dimension(20, 20));
+        }
+        return rigidArea_3;
+    }
+    private Component getRigidArea_4() {
+        if (rigidArea_4 == null) {
+            rigidArea_4 = Box.createRigidArea(new Dimension(20, 20));
+        }
+        return rigidArea_4;
+    }
+   
+    private String infoMessage = "The Shadow Scoreboard Comparison screen displays the current PC2 Shadow and Remote Primary CCS scoreboards, side-by-side."
+            
+            + "\n\nThe comparison between scoreboards starts by considering each RANK (1, 2, 3...)."
+            + "\nFor each rank, all scoreboard rows holding that rank are displayed in each scoreboard."
+            + "\nIf there is a rank present in one scoreboard but not in the other scoreboard (for example, due to ties in earlier ranks),"
+            + "\nan empty row containing question marks is displayed in the scoreboard which does not have an entry for that rank."
+            
+            + "\n\nFor each rank which is present in both scoreboards, if the data for that rank matches in all corresponding fields of the two scoreboards"
+            + "\nthen the row is displayed in green; if any data values do not match then the row is displayed in red."
+            
+            + "\n\nThe effect of the above is that early in a contest when the PC2 Shadow system is still catching up to the Remote CCS, there will typically"
+            + "\nbe many rows where one scoreboard has data that the other does not (because of ties within ranks due to the submissions which that system"
+            + "\nhas/has not judged), and there will be many rows where there is a single rank in both scoreboards but the data values will not match"
+            + "\n(the rows will be red) because the PC2 Shadow system has not judged precisely the same number of submissions as the remote CCS."
+            
+            + "\n\nAt the end of the contest the Scoreboard Comparison screen will show the true comparison of scoreboards for the PC2 Shadow and the Remote CCS."
+            + "\n(If the scoreboards do not match at the end of the contest, the most likely reason is because the PC2 Shadow system has given a different"
+            + "\njudgement than the Remote CCS to one or more submissions; see the PC2 Shadow \"Compare Runs\" grid to find and resolve such differences.)"
+            + "";
```

Comment:
Why is there an empty string at the end of this string, can it be removed ?

Reply:
It was there to make it easier to update the previous lines of text without constantly worrying about making sure the last line of the extended string ends with a semi-colon.  Technically the empty string itself (the double quotes) is not needed -- but the semicolon is.   I removed the double quotes -- although I think it's easier to read when they are present.